### DISCUSSION TITLE
How to use OCR image recognition function to obtain two-dimensional molecular structure？

### DISCUSSION THREAD

**Discussion by ShangChien** [epam/ketcher#1328] [Q&A] [answered]
The function of image recognition is emphasized on github. But after checking the official [documentation](https://lifescience.opensource.epam.com/ketcher/developers-manual.html) there are no examples and any interfaces api for this function. 
Anyone used it? hope for advice，thank you.

**Comment by AndreiMazol** [comment, marked as answer]
Dear @ShangChien ,

sorry for delay. Ketcher API will be extended by this method in scope of release 2.6, issue https://github.com/epam/ketcher/issues/1468:

`ketcher.recognize(blob: Blob, imagoVersion: string)`

Best regards,
Ketcher team

**Reply by wojiaoxiaomayun** [reply]
Dear @AndreiMazol ,
2.6.2 Ketcher API has this,but I read the indigo-service source code and found that the imagoversions of/v2/info have been commented out. return imagoVersion is undefined,please help me.thank you very much.
